packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -391,14 +402,90 @@ - (void)downloadManifestFromURL:(NSURL *)url
   } errorBlock:errorBlock];
 }
 
-- (void)_createUpdateWithManifest:(NSMutableDictionary *)mutableManifest
+- (nullable EXUpdatesCodeSigningConfiguration *)codeSigningConfiguration {
```

Comment:
Might be worth moving this to EXUpdatesConfig since it doesn't seem to depend on this class's state at all. If errors here are solely caused by misconfigured build settings it might also be helpful to throw an uncaught NSException from there, to help developers catch the issue asap. (but of course, not if there's a reasonable chance it could somehow leak out to production without having been caught.)